src/objects/npe01__OppPayment__c.object

```diff
@@ -8,6 +8,96 @@
         <label>NPSP Payment Compact Layout</label>
     </compactLayouts>
     <enableEnhancedLookup>false</enableEnhancedLookup>
+    <fieldSets>
+        <fullName>Payment_Wizard</fullName>
+        <availableFields>
+            <field>npe01__Check_Reference_Number__c</field>
+            <isFieldManaged>false</isFieldManaged>
+            <isRequired>false</isRequired>
+        </availableFields>
+        <availableFields>
+            <field>CreatedById</field>
```

Comment:
Rethinking through this, since the Wizard never shows existing Payment records but only creates new ones, I agree. Removing LastMod By/Date and Created By/Date